src/views/PageError.vue

```diff
@@ -23,7 +23,11 @@ export default {
         },
 
         getMsg() {
-            this.msg = this.$route.params.msg;
+            if (this.$route.params.msg) {
+                this.msg = this.$route.params.msg
+                    .toString()
+                    .replace('Error:', '');
```

Comment:
From where "Error:" is being set in the msg and why.
We can directly remove it from there.

Reply:
throw new Error was written which create Error object 
throw new Error wraps the error passed to it in the following format −

{ name: 'Error', message: 'String you pass in the constructor' } --- quora